dev/fattest.simplicity/src/com/ibm/websphere/simplicity/RemoteFile.java

```diff
@@ -838,17 +1011,18 @@ private static boolean copy(RemoteFile srcFile, RemoteFile destFile,
 
             Log.exiting(c, method, copied);
             return copied;
+
         } else {
             Log.finer(c, method, "The source file is a file. Copying the file.");
             //Now we ensure the parent directory path exists and create if it doesn't as long as it has a parent
-            if (!!!destFile.getParentFile().equals(null)) {
+            if (!destFile.getParentFile().equals(null)) {
```

Comment:
won't we get an NPE here before this statement evaluates to false?
Should be `if (destFile.getParentFile() != null)`

Reply:
Correct to the recommended code.